modules/apps/object/object-rest-test/src/testIntegration/java/com/liferay/object/rest/internal/resource/v1_0/test/ObjectEntryResourceTest.java

```diff
@@ -469,6 +485,86 @@ public void tearDown() throws Exception {
 			_listTypeDefinition);
 	}
 
+	@Test
+	public void testFilterByComparisonOperators() throws Exception {
+		Assert.assertEquals(
+			1,
+			_objectEntryLocalService.getObjectEntriesCount(
+				0, _objectDefinition1.getObjectDefinitionId()));
+
+		DTOConverterContext dtoConverterContext =
+			new DefaultDTOConverterContext(
+				false, Collections.emptyMap(), _dtoConverterRegistry, null,
+				LocaleUtil.getDefault(), null, TestPropsValues.getUser());
+
+		String datetimeObjectFieldValue1 = "2023-12-05T08:50:00.000Z";
+
+		_objectEntryManager.addObjectEntry(
+			dtoConverterContext, _objectDefinition1,
+			new com.liferay.object.rest.dto.v1_0.ObjectEntry() {
+				{
+					properties = Collections.singletonMap(
+						_OBJECT_FIELD_NAME_DATE_TIME,
+						datetimeObjectFieldValue1);
+				}
+			},
+			ObjectDefinitionConstants.SCOPE_COMPANY);
```

Comment:
@devpedrotavares please follow the existing pattern and use the `ObjectEntryTestUtil.addObjectEntry` like in [here](https://github.com/liferay-objects/liferay-portal/blob/2bab6732cc0f6e3b577b30d486b3587dc18ef697/modules/apps/object/object-rest-test/src/testIntegration/java/com/liferay/object/rest/internal/resource/v1_0/test/ObjectEntryResourceTest.java#L3016-L3018).